UPGRADE.md

```diff
@@ -15,6 +15,15 @@ classes for HTML and LaTex, which receive the new `ListNode`.
 Removed `Doctrine\RST\Parser\ListLine` in favor of `Doctrine\RST\Parser\ListItem`
 and changed signature of `Doctrine\RST\Parser\LineChecker::isListLine()`.
 
+## Property visibility changed from protected to private
+
+Some directives and node classes had protected visibility for their properties.
+That has now been changed to private.
+
+## Final classes by default
+
+Many classes have been made final because they were never actually extended.
+
```

Comment:
This probably has to target 0.5, now that 0.4 is released.

Reply:
Oh right, absolutely!